src/blob_storage.h

```diff
@@ -60,6 +66,11 @@ class BlobStorage {
     return files_.size();
   }
 
+  std::size_t NumObsoleteBlobFiles() const {
```

Comment:
Help me add a `TODO` to return directly from this function for the obsolete files DB property?